### ISSUE TITLE
Syntax colors not updating after saving to html from preview after updating atom to 1.13

### ISSUE THREAD

**Issue by raash1d** [atom/markdown-preview#461]
### Prerequisites

* [ x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

I use the save to html feature in markdown-preview to create html files of my notes. The syntax highlighting in the code sections of my markdown used to populate in the html files as well. However, after updating to atom v1.13 and markdown-preview to v0.159.1, there seems to be a breakage in functionality. The syntax highlighting is gone. Fortunately I saved an html created using atom v1.12 and markdown-preview v0.158.8 to show the difference. Please find the images attached below to see the difference. I have also attached the html code with version name of atom with which they were created. Using a simple diff between both files shows many changes to the CSS. This is probably because of [#436](https://github.com/atom/markdown-preview/pull/436)

### Steps to Reproduce

1. Write the following code using atom v1.12 and markdown-preview v0.158.8
```java
String URL = "http://www.DemoQA.com";
driver.get(URL);
driver.get("https://www.google.com");
```
2. Press `ctrl+shift+m` and save to html by right clicking on the preview, open in a browser.
2. Write same code using atom v1.13 and markdown-preview to v0.159.1
3. Perform Step 2.

**Expected behavior:**
Colors of the code should be intact in the html

**Actual behavior:**
Colors are gone, only black text in code.

**Reproduces how often:**
100%

### Versions

Atom    : 1.13.0
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0

### Additional Information
Left side is the markdown code. Right side is the preview in markdown-preview v0.159.1
![atom screenshot](https://cloud.githubusercontent.com/assets/15528027/21958477/7ce2034e-da7d-11e6-81da-c9c93913a9a7.png)

Left side is from atom v1.12 and markdown-preview v0.158.8. Right side is from atom v1.13 and markdown-preview to v0.159.1
![browser-screenshot-difference](https://cloud.githubusercontent.com/assets/15528027/21958487/d8e07306-da7d-11e6-8582-3420fc35e69c.png)

HTML files:
Using atom v1.13 and markdown-preview to v0.159.1:
[get-commands-after-update.txt](https://github.com/atom/markdown-preview/files/706248/get-commands-after-update.txt)

Using atom v1.12 and markdown-preview v0.158.8
[get-commands-before-update.txt](https://github.com/atom/markdown-preview/files/706249/get-commands-before-update.txt)

**Comment by Joinyy** [comment]
+1
used it very often before and now testet several other addons but none of them is as nice as markdown preview before the update

**Comment by ungb** [comment]
@raash1d  Thanks for reporting this.  I was able to confirm this is a regression.  


Issue is the css looks like it was changed, but the class in the html wasn't.  

see below of the css that's generated in the html:

![image](https://cloud.githubusercontent.com/assets/5624870/22035728/eb4434cc-dca5-11e6-94af-f21c4ee3791d.png)

looking at the html they are identical, but if I update the classes to match the css it will work.  Not sure when this broke.

**Comment by raash1d** [comment]
@ungb Yes, when I used the diff checker tool, it showed changes only in the CSS. As mentioned in my report, I believe this is due issue #436. Thanks for looking at this.

**Comment by ungb** [comment]
@as-cii Here are the repro steps I followed for both 1.12.9 and 1.13.0.
1. Open a new file
2. Paste in the following text:
\`\`\`
String URL = "http://www.DemoQA.com";
driver.get(URL);
driver.get("https://www.google.com");
\`\`\`
3. Press ctrl+shift+m and save to html by right clicking on the markdown preview
4. open the saved html in a browser.

On 1.12.9 the code has syntax highlighting.  On 1.13.0 this doesn't work due to the css that is generated.  If you look above on my comment you can see all the css is prefix with `syntax--`, but  the DOM is identical.  If I manually update the classes to have the prefix `syntax--` this works.

Thanks @raash1d for reporting this issue and providing good repro steps.

**Comment by as-cii** [comment]
Thanks for the reports everyone and apologies for this regression. We have just merged a fix for it and it will ship with Atom 1.13.1 and 1.14.0-beta3.
